.travis.yml

```diff
@@ -9,6 +9,7 @@ php:
   - 7.1
```

Comment:
Can we remove [non-maintained php versions](https://www.php.net/supported-versions.php) on that session? We can add other newer versions as well.

Reply:
I also added 7.3